mk/spksrc.configure.mk

```diff
@@ -47,9 +47,8 @@ REAL_CONFIGURE_ARGS += $(CONFIGURE_ARGS)
 
 configure_msg:
 	@$(MSG) "Configuring for $(NAME)"
-	@$(MSG)     - Configure ARGS: $(CONFIGURE_ARGS)
+	@$(MSG)     - Configure ARGS: $(REAL_CONFIGURE_ARGS)
```

Comment:
Really?  isn't REAL_CONFIGURE_ARGS related to GNU_CONFIGURE only?

Reply:
REAL_CONFIGURE_ARGS contains all configure args 

with GNU_CONFIGURE = 1, it contains the --host, --build and --prefix args
and it contains the TC_CONFIGURE_ARGS
plus for DSM 7+ it has --localstatedir defined

without GNU_CONFIGURE it is the same as CONFIGURE_ARGS.


I changed the message to REAL_CONFIGURE_ARGS after I found in cross/gdb-7.12/Makefile there is GNU_CONFIGURE = 1, but finally only CONFIGURE_ARGS are used and the REAL_CONFIGURE_ARGS logic is implemented again.


Is is only confusing with custom configure targets, but this is always the case with a common pre-target message.
Custom configure targets should always add own messages to avoid confusion (when not using REAL_CONDIGURE_ARGS).